src/test/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/resource/ReportResourceTest.kt

```diff
@@ -573,6 +574,18 @@ class ReportResourceTest : SqsIntegrationTestBase() {
               "prisonId": "MDI",
               "title": "Incident Report 11124143",
               "description": "A new incident created in the new service of type find of illicit items",
+              "descriptionAddendums": [
+                {
+                  "createdBy": "staff-1",
```

Comment:
Its worth having their name to make display quicker too

Reply:
done here: https://github.com/ministryofjustice/hmpps-incident-reporting-api/pull/313/commits/3047e4c3a02d81617abe4eda44953f78c04c687a